pkg/metrics/metrics.jsx

```diff
@@ -1322,9 +1328,13 @@ class MetricsHistory extends React.Component {
                 // re-render
                 this.setState({ hours, loading: false });
 
-                // trigger automatic update every minute
-                if (!limit)
-                    this.history_refresh_timer = window.setTimeout(() => this.load_data(this.most_recent), 60000, false);
```

Comment:
What was this last argument `false`?

Reply:
Good question -- the [API](https://developer.mozilla.org/en-US/docs/Web/API/setTimeout) allows additional arguments, but they are just passed to the called function. That lambda does not take any arguments, so that `false` was entirely ignored. Hence I dropped it. I could have mentioned that in the commit message -- I can do that at squash&merge time (unless there's some code change needed)

Reply:
It is fine, don't worry about it. I also figured out it was ignored and was most likely some copy paste relic but wanted to double check.